common/src/main/java/com/wynntils/wynn/model/questbook/QuestBookManager.java

```diff
@@ -80,7 +96,7 @@ private static void processQuestBookPage(ContainerContent container, int page) {
         if (page == 4) {
             // Last page finished
             quests = newQuests;
-            WynntilsMod.postEvent(new QuestBookReloadedEvent());
+            WynntilsMod.getEventBus().post(new QuestBookReloadedEvent.QuestsReloaded());
```

Comment:
This seem like a merge error..? I recently turned `WynntilsMod.getEventBus().post` into `WynntilsMod.postEvent` but this seem to revert that.

Reply:
Yeah I will fix this